libraries/AP_Notify/ExternalLED.h

```diff
@@ -23,6 +23,9 @@
 
 #include "NotifyDevice.h"
 
+#define HIGH 1
+#define LOW 0
```

Comment:
Ok. HIGH/LOW are really bad defines but they are there so I think it's fine to add these ones.